packages/framework/src/Command/CreateApplicationDirectoriesCommand.php

```diff
@@ -95,23 +88,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
      */
     private function createMiscellaneousDirectories(OutputInterface $output)
     {
-        $directories = [
-            '/web/' . $this->webContentDirName . '/feeds',
-            '/web/' . $this->webContentDirName . '/sitemaps',
-            '/web/' . $this->webContentDirName . '/wysiwyg',
-        ];
-
-        $localDirectories = [
-            $this->projectDir . '/build/stats',
-            $this->projectDir . '/docs/generated',
-            $this->projectDir . '/var/cache',
-            $this->projectDir . '/var/lock',
-            $this->projectDir . '/var/logs',
-            $this->projectDir . '/var/errorPages',
-            $this->projectDir . '/web/assets/admin/styles',
-            $this->projectDir . '/web/assets/frontend/styles',
-            $this->projectDir . '/web/assets/scripts',
-        ];
+        $directories = $this->container->getParameter('main_filesystem_directories');
+        $localDirectories = $this->container->getParameter('local_filesystem_directories');
```

Comment:
I'd rather use DI through the constructor than getting the parameters from container directly.